manifests/c/Chatterino/Chatterino/2.3.2/Chatterino.Chatterino.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Chatterino.Chatterino
+PackageVersion: 2.3.2
```

Comment:
```suggestion
PackageVersion: default
```

Reply:
cc @fourtf @DRSchlaubi `PackageVersion` must match the one in Control Panel. If you think this isn't correct then please update your program to change the `PackageVersion` to something else rather than only changing it in the manifest.